### PR TITLE
Update basic publishing rules

### PR DESCRIPTION
**Pull request by brinxmat**
- Clarify basic publishing rules
- Add original doi (the doi in EntityDescription.Reference) as a potential source of validity
- Next step: add isPublishable to types, but this must be done following on from task regarding moving types

### CODE REVIEW

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java [hunk 1]
```diff
@@ -323,11 +324,28 @@ public String getJsonLdContext() {
...
     @JsonIgnore
     public boolean isPublishable() {
-        return !DRAFT_FOR_DELETION.equals(getStatus()) && nonNull(getEntityDescription()) &&
-                !isEmpty(getEntityDescription().getMainTitle()) && getAssociatedArtifacts().isPublishable();
+        return !DRAFT_FOR_DELETION.equals(getStatus()) && hasMainTitle() && hasReferencedContent();
 
     }
 
+    private boolean hasReferencedContent() {
+        return getAssociatedArtifacts().isPublishable() || hasOriginalDoi();
```
Comment (ketilaa) [inline nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java:332]: Is OR correct here? Has DOI anything to do with referenced content?

**Review by ketilaa — APPROVED** [review]

File: nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java (hunk 1 above)
Comment (brinxmat) [reply nva-datamodel-java/src/main/java/no/unit/nva/model/Publication.java:332]: Implies the file is elsewhere

File: nva-datamodel-java/src/test/java/no/unit/nva/PublicationTest.java [hunk 2]
```diff
@@ -277,10 +277,47 @@ private static Publication publicationWithoutEntityDescription() {
...
+
+    private static Publication publicationWithAdministrativeAgreementAndPublishedFile() {
+        var publishedFile = PublishedFileGenerator.random();
+        var administrativeAgreement = AdministrativeAgreementGenerator.random();
+        return publicationWithAssociatedArtifact(new AssociatedArtifactList(List.of(administrativeAgreement,
+                publishedFile)));
+    }
+
+    private static Publication publicationWithOriginalDoi() {
+        var publication = randomPublication();
```
Comment (torarnet) [inline nva-datamodel-java/src/test/java/no/unit/nva/PublicationTest.java:310]: randomPublication has a original DOI?

**Review by torarnet — APPROVED** [review]

File: nva-datamodel-java/src/test/java/no/unit/nva/PublicationTest.java (hunk 2 above)
Comment (brinxmat) [reply nva-datamodel-java/src/test/java/no/unit/nva/PublicationTest.java:310]: Yes, all fields are filled in a randomPublication

**Pull request merged**